syntax = "proto3";
import "product.proto";

message ConfigMessage {
  string name = 1;
  int32 list_id = 2;
  repeated Product products = 3;
}